### PR TITLE
Es6 

### PR DESCRIPTION
**Pull request by Junnplus**

### CODE REVIEW

File: src/views/Cafe.vue [hunk 1]
```diff
@@ -37,67 +37,67 @@
 </template>
 
 <script>
-  var api = require('../api');
+import api from '../../lib/api';
```
Comment (lepture) [inline src/views/Cafe.vue]: why not move api from lib to src?

File: src/views/Cafe.vue [hunk 2]
```diff
@@ -37,67 +37,67 @@
...
 
-  module.exports = {
-    data: function() {
-      var slug = this.$route.params.slug;
-      return {
-        isFollowing: false,
-        cafe: {slug: slug}
-      };
+module.exports = {
```
Comment (lepture) [inline src/views/Cafe.vue]: not ES6 style.

```js
export default {
}
```

File: src/views/Cafe.vue [hunk 3]
```diff
@@ -37,67 +37,67 @@
...
+  data: function() {
```
Comment (lepture) [inline src/views/Cafe.vue]: simple as:

```js
data() {
}
```

File: src/views/Cafe.vue (hunk 1 above)
Comment (Junnplus) [inline src/views/Cafe.vue]: follow you write. modify it right now

**Pull request merged**